app/views/builds/show.html.haml

```diff
@@ -35,10 +35,10 @@
       Create pull request
   %span.info
     Created
-    %abbr.timeago{:title => @build.created_at.to_json} recently
+    %abbr.timeago{:title => @build.created_at.getutc.iso8601}= @build.created_at
```

Comment:
Is there a way to make
  %abbr.timeago{:title => SOMETHING.getutc.iso8601}= SOMETHING
into a function, so we can just have the haml say
 timeThing(@build.created_at)
?
Because it seems like this would be useful in pretty much all other changed places (as well as be a central place to change this further if we want to tweak the UI again)